datafusion/core/tests/parquet/file_statistics.rs

```diff
@@ -198,7 +198,8 @@ fn get_cache_runtime_state() -> (
         .with_list_files_cache(Some(list_file_cache.clone()));
 
     let rt = Arc::new(
-        RuntimeEnv::new(RuntimeConfig::new().with_cache_manager(cache_config)).unwrap(),
+        RuntimeEnv::new(RuntimeEnvBuilder::new().with_cache_manager(cache_config))
```

Comment:
I thought the issue intended to avoid calling `RuntimeEnv::new`, instead have it constructed via the builder pattern:

```rs
let runtime: RuntimeEnv = RuntimeEnvBuilder::new()
    .with_xxx()
    .with_yyy()
    .build()
```